### PR TITLE
add project folder and three new keyboards

### PR DESCRIPTION
**Pull request by TheRoyalSweatshirt**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/the_royal/liminal/config.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1337
```
Comment (drashna) [inline keyboards/the_royal/liminal/config.h]: This vender ID is owned already, please use a different VID. 

https://www.the-sz.com/products/usbid/index.php?v=0x1337&p=&n=

File: keyboards/the_royal/schwann/config.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1337
```
Comment (drashna) [inline keyboards/the_royal/schwann/config.h]: This Vendor ID is owned, please change it.

Comment (TheRoyalSweatshirt) [reply keyboards/the_royal/schwann/config.h]: I never understood that before lol. 

Ive never changed or messed with that config in the past

**Comment by TheRoyalSweatshirt** [comment]
Is there anything else I need to look at?  I think I got everything situated...I hope lol

File: keyboards/the_royal/liminal/config.h [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+#define PERMISSIVE_HOLD
+
+#define TAPPING_TERM 200
+
+#define TAPPING_TERM_PER_KEY
+
```
Comment (zvecr) [inline keyboards/the_royal/liminal/config.h]:
```suggestion

```

File: keyboards/the_royal/liminal/rules.mk [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+TAP_DANCE_ENABLE = yes
```
Comment (zvecr) [inline keyboards/the_royal/liminal/rules.mk]:
```suggestion
```

File: keyboards/the_royal/liminal/rules.mk [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
+
```
Comment (zvecr) [inline keyboards/the_royal/liminal/rules.mk]:
```suggestion

```

File: keyboards/the_royal/schwann/rules.mk [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+TAP_DANCE_ENABLE = yes
```
Comment (zvecr) [inline keyboards/the_royal/schwann/rules.mk]:
```suggestion
```

File: keyboards/the_royal/sm_aio/config.h [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+#define OLED_DISABLE_TIMEOUT
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+#define TAPPING_TERM 200
+
```
Comment (zvecr) [inline keyboards/the_royal/sm_aio/config.h]:
```suggestion

```

File: keyboards/the_royal/sm_aio/config.h [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+#define UNUSED_PINS
+
+#define ENCODERS_PAD_A { B3 }
+#define ENCODERS_PAD_B { B2 }
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+#define OLED_DISABLE_TIMEOUT
+
```
Comment (zvecr) [inline keyboards/the_royal/sm_aio/config.h]:
```suggestion

```

File: keyboards/the_royal/sm_aio/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,125 @@
+#include QMK_KEYBOARD_H
+#include "quantum.h"
+
```
Comment (zvecr) [inline keyboards/the_royal/sm_aio/keymaps/default/keymap.c]:
```suggestion

```

File: keyboards/the_royal/sm_aio/rules.mk [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
+ENCODER_ENABLE = yes        # Enable support for EC11 Rotary Encoder
+OLED_DRIVER_ENABLE = yes    # Enable Support for SSD1306 or SH1106 OLED Displays; Communicating over I2C
+TAP_DANCE_ENABLE = yes
```
Comment (zvecr) [inline keyboards/the_royal/sm_aio/rules.mk]:
```suggestion
```

File: keyboards/the_royal/liminal/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+enum custom_keycodes {
+  M_PAR = SAFE_RANGE,
+  M_BKT
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case M_PAR:
+      if (record->event.pressed) {
+        // when keycode QMKURL is pressed
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/the_royal/liminal/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,165 @@
...
+        SEND_STRING("()" SS_TAP(X_LEFT));
+      } else {
+        // when keycode QMKBEST is released
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/the_royal/liminal/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,165 @@
...
+      }
+      break;
+    case M_BKT:
+      if (record->event.pressed) {
+        // when keycode QMKURL is pressed
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/the_royal/liminal/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,165 @@
...
+        SEND_STRING("[]" SS_TAP(X_LEFT));
+      } else {
+        // when keycode QMKURL is released
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/the_royal/liminal/keymaps/default_iso/keymap.c [hunk 15]
```diff
@@ -0,0 +1,165 @@
...
+enum custom_keycodes {
+  M_PAR = SAFE_RANGE,
+  M_BKT
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case M_PAR:
+      if (record->event.pressed) {
+        // when keycode QMKURL is pressed
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default_iso/keymap.c]:
```suggestion
```

File: keyboards/the_royal/liminal/keymaps/default_iso/keymap.c [hunk 16]
```diff
@@ -0,0 +1,165 @@
...
+        SEND_STRING("()" SS_TAP(X_LEFT));
+      } else {
+        // when keycode QMKBEST is released
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default_iso/keymap.c]:
```suggestion
```

File: keyboards/the_royal/liminal/keymaps/default_iso/keymap.c [hunk 17]
```diff
@@ -0,0 +1,165 @@
...
+      }
+      break;
+    case M_BKT:
+      if (record->event.pressed) {
+        // when keycode QMKURL is pressed
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default_iso/keymap.c]:
```suggestion
```

File: keyboards/the_royal/liminal/keymaps/default_iso/keymap.c [hunk 18]
```diff
@@ -0,0 +1,165 @@
...
+        SEND_STRING("[]" SS_TAP(X_LEFT));
+      } else {
+        // when keycode QMKBEST is released
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default_iso/keymap.c]:
```suggestion
```

File: keyboards/the_royal/liminal/readme.md [hunk 19]
```diff
@@ -0,0 +1,15 @@
...
+
+A 50% keyboard with superb keycap compatibility.
+
+Keyboard Maintainer: [TheRoyalSweatshirt](https://github.com/TheRoyalSweatshirt) a.k.a [The_Royal](https://reddit.com/u/The_Royal)
+Hardware Supported: rev 0.2 Liminal PCB 
+Hardware Availability: Through GB, Proxy, or Direct Message
+
+Make example for this keyboard (after setting up your build environment):
+
+    make the_royal/liminal:deafult
```
Comment (tzarc) [inline keyboards/the_royal/liminal/readme.md]:
```suggestion
    make the_royal/liminal:default
```

File: keyboards/the_royal/sm_aio/keymaps/default/keymap.c [hunk 20]
```diff
@@ -0,0 +1,125 @@
...
+  oled_write_P(qmk_logo, false);
+}
+
+
+extern bool oled_initialized;
+void matrix_scan_user(void) {
+  if(!oled_initialized) {
+    wait_ms(200);
+        oled_init(0);
+
```
Comment (tzarc) [inline keyboards/the_royal/sm_aio/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/the_royal/schwann/config.h [hunk 21]
```diff
@@ -0,0 +1,40 @@
+
+#pragma once
```
Comment (zvecr) [inline keyboards/the_royal/schwann/config.h]:
```suggestion
#pragma once
```

File: keyboards/the_royal/sm_aio/keymaps/default/keymap.c [hunk 22]
```diff
@@ -0,0 +1,125 @@
...
+uint16_t        oled_timer;
+void oled_task_user(void) {
+
+    if (timer_elapsed(oled_timer) > 200000) {
+        oled_off();
+        return;
+    }
+
+            render_logo();
+
```
Comment (tzarc) [inline keyboards/the_royal/sm_aio/keymaps/default/keymap.c]:
```suggestion
    render_logo();
```

File: keyboards/the_royal/liminal/rules.mk [hunk 23]
```diff
@@ -0,0 +1,42 @@
...
+COMMAND_ENABLE = no       # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (zvecr) [inline keyboards/the_royal/liminal/rules.mk]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output on port C6
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```

File: keyboards/the_royal/sm_aio/rules.mk [hunk 24]
```diff
@@ -0,0 +1,25 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
```
Comment (zvecr) [inline keyboards/the_royal/sm_aio/rules.mk]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output on port C6
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```

File: keyboards/the_royal/sm_aio/rules.mk [hunk 25]
```diff
@@ -0,0 +1,25 @@
+MCU = atmega32u4
```
Comment (zvecr) [inline keyboards/the_royal/sm_aio/rules.mk]:
```suggestion
# MCU name
MCU = atmega32u4
```

File: keyboards/the_royal/sm_aio/rules.mk [hunk 26]
```diff
@@ -0,0 +1,25 @@
...
+
+BOOTLOADER = atmel-dfu
```
Comment (zvecr) [inline keyboards/the_royal/sm_aio/rules.mk]:
```suggestion
# Bootloader selection
BOOTLOADER = atmel-dfu
```

File: keyboards/the_royal/sm_aio/readme.md [hunk 27]
```diff
@@ -0,0 +1,16 @@
+
+# SM_A.I.O.
```
Comment (zvecr) [inline keyboards/the_royal/sm_aio/readme.md]:
```suggestion
# SM_A.I.O.
```

File: keyboards/the_royal/liminal/keymaps/default/keymap.c [hunk 28]
```diff
@@ -0,0 +1,165 @@
+#include QMK_KEYBOARD_H
+#include "quantum.h"
+
```
Comment (zvecr) [inline keyboards/the_royal/liminal/keymaps/default/keymap.c]:
```suggestion

```

File: keyboards/the_royal/liminal/keymaps/default_iso/keymap.c [hunk 29]
```diff
@@ -0,0 +1,165 @@
+#include QMK_KEYBOARD_H
+#include "quantum.h"
+
```
Comment (zvecr) [inline keyboards/the_royal/liminal/keymaps/default_iso/keymap.c]:
```suggestion

```

File: keyboards/the_royal/schwann/keymaps/default/keymap.c [hunk 30]
```diff
@@ -0,0 +1,137 @@
+#include QMK_KEYBOARD_H
+#include "quantum.h"
+
```
Comment (zvecr) [inline keyboards/the_royal/schwann/keymaps/default/keymap.c]:
```suggestion

```

File: keyboards/the_royal/liminal/keymaps/default/keymap.c [hunk 31]
```diff
@@ -0,0 +1,165 @@
...
+        TD(TD_RST),  XXXXX,      XXXXX,      XXXXX,                            XXXXX,                    XXXXX,       XXXXX,       XXXXX
+    )
+
+};
+
+uint16_t get_tapping_term(uint16_t keycode) {
+  switch (keycode) {
+    case LT(_LOWER, KC_SPACE):
+      return 160;
+    case TD(TD_DBQT):
```
Comment (drashna) [inline keyboards/the_royal/liminal/keymaps/default/keymap.c]: Just a heads up, this won't actually work, at this time.   Not until #6259 is merged.

File: keyboards/the_royal/liminal/config.h (hunk 3 above)
Comment (TheRoyalSweatshirt) [reply keyboards/the_royal/liminal/config.h]: yeah Ill change it.  Ive always just like XXX or XXXX cause it helps keep the layers pretty even in terms of physical spacing

File: keyboards/the_royal/liminal/keymaps/default/keymap.c (hunk 31 above)
Comment (TheRoyalSweatshirt) [reply keyboards/the_royal/liminal/keymaps/default/keymap.c]: Its working right now....322155342 lol Did I mess something up

Comment (TheRoyalSweatshirt) [reply keyboards/the_royal/liminal/keymaps/default/keymap.c]: or you mean the timing?

File: keyboards/the_royal/liminal/rules.mk (hunk 4 above)
Comment (TheRoyalSweatshirt) [reply keyboards/the_royal/liminal/rules.mk]: So you want these values to all be consolidated into the Keymap correct?

File: keyboards/the_royal/liminal/keymaps/default/keymap.c (hunk 31 above)
Comment (TheRoyalSweatshirt) [reply keyboards/the_royal/liminal/keymaps/default/keymap.c]: Oh okay I see.  Should I remove it or will it just be default until #6259 is merged?

**Comment by TheRoyalSweatshirt** [comment]
I don't know why the submodules changed.  Is there a specific way to go about setting that up before I commit changes.

File: keyboards/the_royal/liminal/readme.md [hunk 32]
```diff
@@ -0,0 +1,15 @@
+# Liminal
+====
+![liminal_ec](https://i.imgur.com/9Xe7uEa.jpg)
+
+A 50% keyboard with superb keycap compatibility.
+
+Keyboard Maintainer: [TheRoyalSweatshirt](https://github.com/TheRoyalSweatshirt) a.k.a [The_Royal](https://reddit.com/u/The_Royal)
+Hardware Supported: rev 0.2 Liminal PCB 
+Hardware Availability: Through GB, Proxy, or Direct Message
```
Comment (fauxpark) [inline keyboards/the_royal/liminal/readme.md:9]:
```suggestion
* Keyboard Maintainer: [TheRoyalSweatshirt](https://github.com/TheRoyalSweatshirt) a.k.a [The_Royal](https://reddit.com/u/The_Royal)
* Hardware Supported: rev 0.2 Liminal PCB
* Hardware Availability: Through GB, Proxy, or Direct Message
```

File: keyboards/the_royal/liminal/readme.md [hunk 33]
```diff
@@ -0,0 +1,15 @@
+# Liminal
+====
```
Comment (fauxpark) [inline keyboards/the_royal/liminal/readme.md:2]:
```suggestion

```

File: keyboards/the_royal/liminal/rules.mk [hunk 34]
```diff
@@ -0,0 +1,33 @@
+# MCU name
+MCU = atmega32u2
+
+
```
Comment (fauxpark) [inline keyboards/the_royal/liminal/rules.mk:4]:
```suggestion
```

File: keyboards/the_royal/liminal/rules.mk [hunk 35]
```diff
@@ -0,0 +1,33 @@
...
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
```
Comment (fauxpark) [inline keyboards/the_royal/liminal/rules.mk:11]:
```suggestion
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

**Comment by TheRoyalSweatshirt** [comment]
Okay...I think shes ready

File: keyboards/the_royal/schwann/readme.md [hunk 36]
```diff
@@ -0,0 +1,16 @@
+# Schwann PCB
+====
```
Comment (fauxpark) [inline keyboards/the_royal/schwann/readme.md:2]:
```suggestion
```

File: keyboards/the_royal/schwann/readme.md [hunk 37]
```diff
@@ -0,0 +1,16 @@
...
+
+![schwann_render](https://i.imgur.com/04uaN8x.jpg)
+
+A simple, HHKB ortho PCB, designed for use with the Neuron Keyboard by Wallet Burner
+
+Keyboard Maintainer: [TheRoyalSweatshirt](https://github.com/TheRoyalSweatshirt) a.k.a [The_Royal](https://reddit.com/u/The_Royal)
+Hardware Supported: Rev0.2 and Rev0.5 Schwann PCB 
+Hardware Availability: Private Message, Group buy, or self-procurement via the open sourced [Schwann Documentation](https://github.com/TheRoyalSweatshirt/The_Royal_Projects/tree/schwann/PCBs/Schwann_PCB).
```
Comment (fauxpark) [inline keyboards/the_royal/schwann/readme.md:10]:
```suggestion
* Keyboard Maintainer: [TheRoyalSweatshirt](https://github.com/TheRoyalSweatshirt) a.k.a [The_Royal](https://reddit.com/u/The_Royal)
* Hardware Supported: Rev0.2 and Rev0.5 Schwann PCB
* Hardware Availability: Private Message, Group buy, or self-procurement via the open sourced [Schwann Documentation](https://github.com/TheRoyalSweatshirt/The_Royal_Projects/tree/schwann/PCBs/Schwann_PCB).
```

File: keyboards/the_royal/schwann/rules.mk [hunk 38]
```diff
@@ -0,0 +1,31 @@
+# MCU name
+MCU = atmega32u4
+
+
```
Comment (fauxpark) [inline keyboards/the_royal/schwann/rules.mk:4]:
```suggestion
```

File: keyboards/the_royal/schwann/rules.mk [hunk 39]
```diff
@@ -0,0 +1,31 @@
...
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
```
Comment (fauxpark) [inline keyboards/the_royal/schwann/rules.mk:11]:
```suggestion
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

File: keyboards/the_royal/schwann/rules.mk [hunk 40]
```diff
@@ -0,0 +1,31 @@
...
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE = no        # Console for debug(+400)
+COMMAND_ENABLE = no       # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
```
Comment (fauxpark) [inline keyboards/the_royal/schwann/rules.mk:27]:
```suggestion
BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no        # Console for debug
COMMAND_ENABLE = no       # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
```

File: keyboards/the_royal/sm_aio/keymaps/default/keymap.c [hunk 41]
```diff
@@ -0,0 +1,126 @@
...
+     * │Home│ │Esc │ Q  │ W  │ E  │ R  │ T  │ Y  │ U  │ I  │ O  │ P  │ -  │BkSp │
+     * ├────┤ ├────┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴─────┤
+     * │PgUp│ │Tab  │ A  │ S  │ D  │ F  │ G  │ H  │ J  │ K  │ L  │ ;  │  Enter  │
+     * ├────┤ ├─────┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬───────┤
+     * │PgDn│ │L_Shft │ Z  │ X  │ C  │ V  │ B  │ N  │ M  │ ,  │ .  │ /  │R_Shft │
+     * ├────┤ ├──────┬┴────┼────┴─┬──┴────┴───┬┴────┴────┴──┬─┴────┼────┴─┬─────┤
+     * │END │ │L_Ctrl│L_GUI│L_Alt │    FN     │    SPACE    │R_Ctrl│R_GUI │R_Alt│
+     * └────┘ └──────┴─────┴──────┴───────────┴─────────────┴──────┴──────┴─────┘
+*/
+[_BASE] = LAYOUT_split_space( \
```
Comment (fauxpark) [inline keyboards/the_royal/sm_aio/keymaps/default/keymap.c:51]:
```suggestion
[_BASE] = LAYOUT_split_space(
```

File: keyboards/the_royal/sm_aio/keymaps/default/keymap.c [hunk 42]
```diff
@@ -0,0 +1,126 @@
...
+     * ├────┤ ├────┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴─────┤
+     * │ ~  │ │Tab  │ !  │ @  │ #  │ $  │ %  │ ^  │ &  │ *  │ (  │ )  │         │
+     * ├────┤ ├─────┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬───────┤
+     * │ │  │ │L_Shft │    │    │    │    │    │    │    │    │ [  │  ] │       │
+     * ├────┤ ├──────┬┴────┼────┴─┬──┴────┴───┬┴────┴────┴──┬─┴────┼────┴─┬─────┤
+     * │ \  │ │L_Ctrl│L_GUI│L_Alt │           │             │      │      │     │
+     * └────┘ └──────┴─────┴──────┴───────────┴─────────────┴──────┴──────┴─────┘
+*/
+
+[_FN] = LAYOUT_split_space( \
```
Comment (fauxpark) [inline keyboards/the_royal/sm_aio/keymaps/default/keymap.c:74]:
```suggestion
[_FN] = LAYOUT_split_space(
```

File: keyboards/the_royal/sm_aio/readme.md [hunk 43]
```diff
@@ -0,0 +1,15 @@
...
+
+![AIO_SMKeyboard_case](https://i.imgur.com/OjIBJeS.jpg)
+
+A commissioned, All-In-One 50% Keyboard PCB, designed and fabricated for Sam of SMKeyboards.
+
+- Keyboard Maintainer: [TheRoyalSweatshirt](https://github.com/TheRoyalSweatshirt) a.k.a [The_Royal](https://reddit.com/u/The_Royal)
+- Hardware Supported: aio rev 0.3 & rev 1.2 PCBs
+- Hardware Availability: Through Commission or Private Sale
```
Comment (fauxpark) [inline keyboards/the_royal/sm_aio/readme.md:9]:
```suggestion
* Keyboard Maintainer: [TheRoyalSweatshirt](https://github.com/TheRoyalSweatshirt) a.k.a [The_Royal](https://reddit.com/u/The_Royal)
* Hardware Supported: aio rev 0.3 & rev 1.2 PCBs
* Hardware Availability: Through Commission or Private Sale
```

File: keyboards/the_royal/sm_aio/rules.mk [hunk 44]
```diff
@@ -0,0 +1,31 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
```
Comment (fauxpark) [inline keyboards/the_royal/sm_aio/rules.mk:10]:
```suggestion
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

File: keyboards/the_royal/sm_aio/sm_aio_oled.c [hunk 45]
```diff
@@ -0,0 +1,241 @@
+#pragma once
+
+#ifdef __AVR__
+#    include <avr/io.h>
+#    include <avr/pgmspace.h>
+#elif defined(ESP8266)
+#    include <pgmspace.h>
+#else
+#    define PROGMEM
+#endif
```
Comment (fauxpark) [inline keyboards/the_royal/sm_aio/sm_aio_oled.c:10]:
```suggestion
#include "progmem.h"
```

File: keyboards/the_royal/liminal/keymaps/default/keymap.c [hunk 46]
```diff
@@ -0,0 +1,165 @@
...
+    _LOWER,
+    _RAISE,
+    _OTHER
+};
+
+#define PERMISSIVE_HOLD
+
+#define TAPPING_TERM 200
+
+#define TAPPING_TERM_PER_KEY
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default/keymap.c:14]: these won't have any effect -- they'll need to go into a new `config.h` in the keymap directory

File: keyboards/the_royal/liminal/keymaps/default_iso/keymap.c [hunk 47]
```diff
@@ -0,0 +1,166 @@
...
+    _LOWER,
+    _RAISE,
+    _OTHER
+};
+
+#define PERMISSIVE_HOLD
+
+#define TAPPING_TERM 200
+
+#define TAPPING_TERM_PER_KEY
```
Comment (tzarc) [inline keyboards/the_royal/liminal/keymaps/default_iso/keymap.c:14]: Same here

File: keyboards/the_royal/sm_aio/keymaps/default/keymap.c [hunk 48]
```diff
@@ -0,0 +1,126 @@
...
+// Layer names
+enum{
+  // - Base layer:
+  _BASE,
+  // - Symbols, numbers, and functions:
+  _FN
+};
+
+#define OLED_DISABLE_TIMEOUT
+#define TAPPING_TERM 200
```
Comment (tzarc) [inline keyboards/the_royal/sm_aio/keymaps/default/keymap.c:12]: These too

**Comment by TheRoyalSweatshirt** [comment]
🤞

File: keyboards/the_royal/liminal/keymaps/default/keymap.c (hunk 31 above)
Comment (drashna) [reply keyboards/the_royal/liminal/keymaps/default/keymap.c:137]: It should be fine to leave in here.

**Comment by tzarc** [comment]
Why'd this get closed? Preparing a followup?

**Comment by tzarc** [comment]
@TheRoyalSweatshirt

**Comment by TheRoyalSweatshirt** [comment]
@tzarc yeah I have a few other boards I’m wanting to add and since this one was still pending the merge I thought I might as well get them together when the new boards arrive from the PCB manu and some go public. 

I appreciate all the help!

**Pull request closed without merging**